doc/train-fitting-tensor.md

```diff
@@ -0,0 +1,125 @@
+
+Unlike `energy` which is a scalar, one may want to fit some high dimensional physical quantity, like `dipole` (vector) and `polarizability` (matrix, shorted as `polar`). Deep Potential has provided different API to allow this. In this example we will show you how to train a model to fit them for a water system. A complete training input script of the examples can be found in 
+
+```bash
+$deepmd_source_dir/examples/water_tensor/dipole/dipole_input.json
+$deepmd_source_dir/examples/water_tensor/polar/polar_input.json
+```
+
+The training and validation data are also provided our examples. But note that **the data provided along with the examples are of limited amount, and should not be used to train a productive model.**
+
+
+
+The directory of this examples:
+
+-   [The training input script](# the-training-input-script)
+-   [Training Data Preparation](# training-data-preparation)
+- 	[Train the Model](# train-the-model)
+
+## The training input script
+
+Similar to the `input.json` used in `ener` mode, training json is also divided into `model`, `learning_rate`, `loss` and `training`. Most keywords remains the same as `ener` mode, and their meaning can be found [here](train-se-e2-a.md). To fit a tensor, one need to modify `model.fitting_net` and `loss`.
+
+### Model
+
+The `fitting_net` section tells DP which fitting net to use.
+
+The json of `dipole` type should be provided like
+
+```json
+	"fitting_net" : {
+	    "type": "dipole",
+		"sel_type": [0],
+		"neuron": [100,100,100],
+		"resnet_dt": true,
+		"seed": 1,
+	},
+```
+
+The json of `polar` type should be provided like
+
+```json
+"fitting_net" : {
+    "type": "polar",
+	"sel_type": [0],
+	"neuron": [100,100,100],
+	"resnet_dt": true,
+	"seed": 1,
+},
+```
+
+-   `type` specifies which type of fitting net should be used. It should be either `dipole` or `polar`. Note that `global_polar` mode in version 1.x is already **deprecated** and is merged into `polar`. To specify whether a system is global or atomic, please see [here](train-se-e2-a.md).
+-   `sel_type` is a list specifying which type of atoms have the quantity you want to fit. For example, in water system, `sel_type` is `[0]` since `0` represents for atom `O`. If left unset, all type of atoms will be fitted.
+-   The rest `args` has the same meaning as they do in `ener` mode.
+
+### Loss
+
+DP supports a combinational training of global system (only a global `tensor` label, i.e. dipole or polar, is provided in a frame) and atomic system (labels for **each** atom included in `sel_type` are provided). In a global system, each frame has just **one** `tensor` label. For example, when fitting `polar`, each frame will just provide a `1 x 9` vector indicating the summation of polarizability. By contrast, in a atomic system, each atom in `sel_type` has a `tensor` label. For example, when fitting dipole, each frame will provide a `#sel_atom x 3` matrix, where `#sel_atom` is the number of atoms whose type are in `sel_type`.

```

Comment:
"... provide a `1 x 9` vector indicating the summation of polarizability" -> 
"... provide a `1 x 9` vector which gives the elements of the polarizability tensor of that frame in order XX, XY, XZ, YX, YY, YZ, XZ, ZY, ZZ."